README.md

```diff
@@ -376,48 +376,38 @@ Under the root directory of the ISIS Data Area pointed to by the ISISDATA/ISIS3D
 
 ### Versions of the ISIS Data Area
 
-In ISIS version 4.1.0, several files previously stored in the data area closely associated with ISIS applications were moved into version control with the ISIS source code. Additionally, the environment variables used for the ISIS Data Area and the rsync location for the ISIS Data Area were also updated.
+In ISIS version 4.1.0 and later, several files previously stored in the data area closely associated with ISIS applications were moved into version control with the ISIS source code. To support the use of data in ISIS versions predating 4.1.0 the `downloadIsisData` application will need to be supplied the optional parameter `--legacy`. This is explained further in the [Full ISIS Data Download](README.md#Full-ISIS-Data-Download) section. 
```

Comment:
Should have made the change myself in the previous PR. But for the sake of simplicity I removed the legacy flag and just figured users who want the old base data stuff can download them but explicitly downloading the `legacybase_usgs` target.